tests/test_selectors.py

```diff
@@ -106,3 +106,30 @@ def test_connection_error(self):
             get_bug.side_effect = requests.exceptions.ConnectionError
             with self.assertWarns(RuntimeWarning):
                 selectors.bug_is_fixed(None, ver)
+
+
+class SkipIfConditionTestCase(unittest.TestCase):
+    """Test :meth: `pulp_smash.selectors.skip_if`."""
+
+    class _DummyClass:  # pylint:disable=missing-docstring,too-few-public-methods
+
+        def __init__(self):
+            self.element = 'test'
+
+        @selectors.skip_if(bool, 'element', True, unittest.SkipTest)
+        def dummy_should_skip(self):  # pylint:disable=missing-docstring,no-self-use
+            pass
+
+        @selectors.skip_if(bool, 'element', False, unittest.SkipTest)
+        def dummy_should_run(self, bool_val):  # pylint:disable=missing-docstring,no-self-use
```

Comment:
Why does this method need to accept a `bool_val` argument? So that the `test_skip_false` method can make an assertion, right? I understand, but having an assertion just for having an assertion isn't helpful. Sometimes, the mere fact that a method executes is itself the test.